source/draft_api.rst

```diff
@@ -216,62 +233,135 @@ GET Endpoints
+TimeSync's response data can be narrowed even further than the /:endpoints
+return statements by adding parameters.
+
+==== ======= ======== ========== ================ ===============
+user project activity date range object revisions deleted objects
+==== ======= ======== ========== ================ ===============
+
+Reference Table
+~~~~~~~~~~~~~~~
+
+=================== ======================= =======================
+Parameter           Value(s)                Endpoint(s)
+=================== ======================= =======================
+?user=              :username               /times
+?project=           :projectslug            /times
+?activity=          :activityslug           /times
+?start=             :date (iso format)      /times
+?end=               :date (iso format)      /times
+?revisions=         :bool                   - /times
+                                            - /times/:uuid
+                                            - /activities/
+                                            - /activities/:slug
+                                            - /projects/
+                                            - /projects/:slug
+include_deleted     :bool                   - /times
+                                            - /times/:uuid
+                                            - /activities
+                                            - /projects
+=================== ======================= =======================
+
+?user=:username
+~~~~~~~~~~~~~~~
+
+``/times?user=:username``
+    Filters results to a set of time submitted entries by a specified user.
+
+?project=:projectslug
+~~~~~~~~~~~~~~~~~~~~~
+
+``/times?project=:projectslug``
+    Filters results to a set of time entries of a specified project slug.
+
+?activity=:activityslug
+~~~~~~~~~~~~~~~~~~~~~~~
+
+``/times?activity=:activityslug``
+    Filters results to a set of time entries with a specified activity slug.
+
+?start=:date
+~~~~~~~~~~~~
+
+``/times?start=:date``
+    Filters results to a set of time entries after a specified date.
+
+``/times?end=:date&start=:date``
+    Can be combined with ?end to create a date range.
+
+?end=:date
+~~~~~~~~~~
+
+``/times?end=:date``
+    Filters results to a set of time entries before a specified date.
+
+``/times?start=:date&end=:date``
+    Can be combined with ?start to create a date range.
+
+?include_revisions=:bool
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+``/times?include_revisions=:bool``
+    * Adds the 'parents' field to the specified object.
+    * This field is a list of all previous revisions of the object in
+      descending order by revision number (i.e. ``time.parents[0]`` will be the
+      previous revision, and ``time.parents[n-1]`` will be the first revision).
+    * Without this field the object(s) do not include a 'parents' field and so
+      only the most recent revision of the object will be seen.
+
+?include_deleted=:bool
+~~~~~~~~~~~~~~~~~~~~~~
+
+Includes deleted entries in the returned results.
```

Comment:
That is how restructured text renders things of the
```
word or phrase
    definition of word or phrase
```
So it's not 100% intentional, but that structure is intentional since I am elaborating on a statement with a definition.